src/main/kotlin/com/github/arnaudj/linkify/spi/jira/restclient/Jira7RestClientImpl.kt

```diff
@@ -32,7 +33,9 @@ open class Jira7RestClientImpl(configMap: Map<String, Any>) : JiraRestClient {
                     .addHeader("Authorization", Credentials.basic(jiraAuthUser, jiraAuthPwd))
```

Comment:
to test: Ideally, in case we have a valid auth cookie, I think we should not send Authorization header (perhaps jira will handle authentication rules gracefully and favor cookie if present)